pipelines/broad/dna_seq/germline/variant_calling/VariantCalling.wdl

```diff
@@ -113,6 +115,18 @@ workflow VariantCalling {
       preemptible_tries = agg_preemptible_tries
   }
 
+  if (make_gvcf) {
```

Comment:
@ldgauthier can we update this variable name? The pipeline still outputs a gvcf even if this is false, right? It's just not reblocked. So maybe the variable can be named `reblock_gvcf` or `reblock` ? Still default to true in the pipeline.

Reply:
We probably need to update the changelogs.  This should stay as it is -- if we're making a GVCF then always reblock it, but if we're outputting a genotyped GVCF (no ref blocks, no non-ref alleles) then don't reblock it because that shouldn't really do anything significant.

Reply:
I see - TIL a genotyped GVCF is different than a GVCF! Thanks!